eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -3,61 +3,23 @@ parameters:
   type: boolean
 - name: NuGetLocalizationType
   type: string
-
 steps:
-- task: MicroBuildLocalizationPlugin@1
-  displayName: "Install Localization Plugin"
-
-- task: MicroBuildSigningPlugin@1
-  inputs:
-    signType: "$(SigningType)"
-    esrpSigning: "true"
-  displayName: "Install Signing Plugin"
-
-- task: MicroBuildSwixPlugin@4
-  displayName: "Install Swix Plugin"
-
-- task: MicroBuildOptProfPlugin@6
-  displayName: 'OptProfV2:  install the plugin'
-  inputs:
-    getDropNameByDrop: true
-    optimizationInputsDropNamePrefix: OptimizationInputs/$(System.TeamProject)/$(Build.Repository.Name)
-    ShouldSkipOptimize: $(ShouldSkipOptimize)
-    AccessToken: $(System.AccessToken)
-  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
-
 - task: PowerShell@1
   inputs:
     scriptName: "$(Build.Repository.LocalPath)\\configure.ps1"
     arguments: "-Force"
   displayName: "Run Configure.ps1"
-
 - task: PowerShell@1
   inputs:
     scriptName: "$(Build.Repository.LocalPath)\\scripts\\cibuild\\ConfigureVstsBuild.ps1"
     arguments: "-BuildRTM $(BuildRTM) -RepositoryPath $(Build.Repository.LocalPath) -BranchName $(Build.SourceBranchName) -CommitHash $(Build.SourceVersion) -BuildNumber $(Build.BuildNumber)"
   displayName: "Configure VSTS CI Environment"
-
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish buildinfo.json as an artifact'
-  inputs:
-    ArtifactName: 'BuildInfo'
-    ArtifactType: 'Container'
-    PathToPublish: '$(Build.Repository.LocalPath)\artifacts\buildinfo.json'
-  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'))"
-
 - task: PowerShell@1
   displayName: "Print Environment Variables"
   inputs:
     scriptType: "inlineScript"
     inlineScript: |
       Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
-
-# NuGet.Client's official repo is on github, hence the source link metadata should use the github URL.
```

Comment:
Why remove this? 
What we're doing is unusual and the description helps understanding.

Reply:
Migration tool did it. Just found a few cases of lost comments and put them back.